nilearn/decoding/space_net.py

```diff
@@ -636,12 +658,24 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
          Each pair are are the list of indices for the train and test
          samples for the corresponding fold.
 
-    `cv_scores_` : ndarray, shape (n_alphas, n_folds) or (n_l1_ratios, n_alphas, n_folds)
+    `cv_scores_` : ndarray, shape (n_folds, n_alphas) or (n_l1_ratios, n_folds, n_alphas)
         Scores (misclassification) for each alpha, and on each fold
 
     `screening_percentile_` : float
         Screening percentile corrected according to volume of mask,
         relative to the volume of standard brain.
+
+    `w_` : ndarray, shape (n_classes-1, n_features) #TODO: check shapes
```

Comment:
n_classes-1 is only true for 2 classes? Does this really also hold for one-versus-rest scenarios?